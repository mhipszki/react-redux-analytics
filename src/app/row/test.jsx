import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Row from './';

test('styles a div element as a flexbox row', () => {
  const component = shallow(<Row />);
  expect(component).toMatchSnapshot();
});
