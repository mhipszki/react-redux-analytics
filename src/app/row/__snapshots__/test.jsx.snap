// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles a div element as a flexbox row 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
}

<div
  className="c0"
/>
`;
